app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -227,112 +263,43 @@ class Identification extends React.Component {
                 <div className="nav-input-label">LANGUAGE</div>
               </Form.Row>
               <Form.Row className="pb-3 pt-1 ml-0">
-                Primary Language is used to determine the translations for what the monitoree sees/hears. Currently supported languages are indicated by a * in
-                the below list. When a primary contact method is selected for a language that is not supported, Sara Alert will default to English.
+                Primary Language is used to determine the translations for what the monitoree sees/hears. Languages that are not fully supported are indicated
+                by a * in the below list. When a primary contact method is selected for a language that is not supported, Sara Alert will default to English.
```

Comment:
my original wording here wasn't great... suggest changing `by a * in` to `by an asterisk (*) in`